05-descriptive-analysis.Rmd

```diff
@@ -295,8 +297,8 @@ To calculate a population count estimate with `survey_total()`, we can run the f
 
 ```{r}
 recs_des %>%
-  summarize(survey_total(),
-            .groups = "drop")
+  summarize(survey_total(),  
+            .groups = "drop")  
 ```
 
 Note that the result from `recs_des %>% summarize(survey_total(), .groups = "drop")` is equivalent to the `survey_count()` call. However, the `survey_total()` function is called within `summarize`, where as `survey_count()`, like `dplyr::count()`, is not.
```

Comment:
No groups are used so not sure why we have .groups="drop". Move discussion elsewhere to when group is used.
```suggestion
  summarize(survey_total())  
```

Note that the result from `recs_des %>% summarize(survey_total(), .groups = "drop")` is equivalent to the `survey_count()` call. However, the `survey_total()` function is called within `summarize`, where as `survey_count()`, like `dplyr::count()`, is not.
```